packages/dashboard/src/pages/Labs/components/SwapSettingDialog.tsx

```diff
@@ -80,6 +83,12 @@ export default function SwapSettingDialog({ open, onClose }: SettingDialogProps)
             options: xDaiOptions,
             onChange: (value: any) => Services.Settings.setxDaiNetworkTradeProvider(+value),
         },
+        {
+            legend: t.labs_settings_swap_network({ network: 'Celo' }),
+            value: useCeloNetworkTradeProvider(),
+            options: celoOptions,
+            onChange: (value: any) => Services.Settings.setCeloNetworkTradeProvider(+value),
```

Comment:
plz do not use `any` type, and do not use plus symbol convert unknown type to number type. thanks

Reply:
Hi @septs, should I switch to `number` for the other items in the list too in this commit?

Reply:
CC @guanbinrui

Reply:
Hi again @septs @guanbinrui. May I please get an update on this?

Reply:
@septs @guanbinrui I have changed to `string` since the value being accessed is `event.target.value` in the RadioGroup and based on[ mui docs, that's a string ](https://mui.com/api/radio-group/) and then using `Number(value)` to convert to number.